addons/web/static/src/js/views/calendar/calendar_renderer.js

```diff
@@ -171,6 +171,7 @@ return AbstractRenderer.extend({
     template: "CalendarView",
     config: {
         CalendarPopover: CalendarPopover,
+        calendarBoxTemplate: 'calendar-box',
```

Comment:
I'm debating if we should rename this "eventTemplate".

"calendarBox" doesn't make much sense IMO.

What do you think?